src/auth/tenant.ts

```diff
@@ -253,6 +268,11 @@ export class Tenant {
     if (typeof request.smsRegionConfig != 'undefined') {
       SmsRegionsAuthConfig.validate(request.smsRegionConfig);
     }
+    // Validate passwordPolicyConfig type if provided.
+    if (typeof request.passwordPolicyConfig !== 'undefined') {
+      // This will throw an error if invalid.
+      PasswordPolicyAuthConfig.buildServerRequest(request.passwordPolicyConfig);
```

Comment:
format

Reply:
Not sure I understand? Could make a suggestion maybe?

Reply:
Interesting, seems the format was off in vs code.